src/installer/pkg/packaging/deb/dotnet-runtime-deps-debian_config.json

```diff
@@ -1,12 +1,12 @@
 {
-    "maintainer_name":".NET Core Team",
+    "maintainer_name":".NET Team",
     "maintainer_email": "dotnetpackages@dotnetfoundation.org",
```

Comment:
The email is for `dotnetpackages@dotnetfoundation.org` and the name is `.NET Team`.  Should the name be `.NET Foundation` or something like that?

Reply:
This was introduced in https://github.com/dotnet/core-setup/pull/2525/ . @richlander Is there anything to fix?

Reply:
".NET Team" looks good to me as the "maintainer".